build.gradle

```diff
@@ -224,7 +226,7 @@ dependencies {
     compile group: 'org.awaitility', name: 'awaitility', version: '3.1.6'
 
     // CVE-2019-12086 - force update of jackson-databind - remove when dependencies move off <2.9.9
-    compile "com.fasterxml.jackson.core:jackson-databind:2.9.10"
+    compile "com.fasterxml.jackson.core:jackson-databind:2.9.10.1"
```

Comment:
may be bump to latest version 2.10.0?

Reply:
I'm advised to keep the version delta as small as possible to minimise the scope of regression testing required whilst resolving outstanding CVEs - we should tackle this in a follow up.